iptables/iptables.go

```diff
@@ -473,3 +490,24 @@ func (ipt *IPTables) existsForOldIptables(table, chain string, rulespec []string
 	}
 	return strings.Contains(stdout.String(), rs), nil
 }
+
+// counterRegex is the regex used to detect nftables counter format
+var counterRegex = regexp.MustCompile(`^\[([0-9]+):([0-9]+)\] `)
+
+// filterRuleOutput works around some inconsistencies in output.
+// For example, when iptables is in legacy vs. nftables mode, it produces
+// different results.
+func filterRuleOutput(rule string) string {
+	// work around an output difference in nftables mode where counters
+	// are output in iptables-save format, rather than iptables -S format
+	// The string begins with "[0:0]"
+	//
+	// Fixes #49
+	if groups := counterRegex.FindStringSubmatch(rule); groups != nil {
+		// drop the brackets
+		rule = rule[len(groups[0]):]
+		rule = fmt.Sprintf("%s -c %s %s", rule, groups[1], groups[2])
```

Comment:
Nit: I'd personally prefer not shadowing input parameters and local variables, if possible.